package.json

```diff
@@ -30,41 +28,35 @@
         "zed:production": "node ./node_modules/@spryker/oryx-for-zed/build --prod"
     },
     "engines": {
-        "node": ">=12.0.0"
+        "node": ">=16.0.0",
+        "npm": ">=8.0.0"
     },
     "dependencies": {
         "@webcomponents/custom-elements": "~1.3.1",
         "@webcomponents/webcomponents-platform": "~1.0.1",
         "@webcomponents/webcomponentsjs": "~2.4.0",
-        "classlist-polyfill": "~1.2.0",
         "core-js": "~3.4.1",
-        "element-closest": "~3.0.0",
-        "element-remove": "~1.0.4",
-        "intersection-observer": "~0.10.0",
         "jquery": "~3.5.1",
         "jquery-datetimepicker": "~2.5.21",
         "lodash-es": "~4.17.15",
         "nouislider": "~11.1.0",
         "password-validator": "~5.1.0",
         "select2": "~4.0.11",
-        "slick-carousel": "~1.8.1",
-        "string.prototype.repeat": "~0.2.0",
-        "string.prototype.startswith": "~0.2.0",
-        "whatwg-fetch": "~3.4.0"
+        "slick-carousel": "~1.8.1"
     },
     "devDependencies": {
-        "@babel/plugin-proposal-class-properties": "~7.10.4",
-        "@babel/plugin-transform-runtime": "~7.11.5",
-        "@babel/preset-typescript": "~7.10.4",
+        "@babel/plugin-proposal-class-properties": "~7.17.12",
+        "@babel/plugin-transform-runtime": "~7.17.12",
+        "@babel/preset-typescript": "~7.17.12",
         "@jsdevtools/file-path-filter": "~3.0.2",
         "@spryker/frontend-config.eslint": "~0.0.2",
```

Comment:
devDependencies `"prettier": "~2.5.1"` and `"@prettier/plugin-xml": "~0.13.1"` are missed.